site.yml

```diff
@@ -60,11 +60,11 @@ asciidoc:
     occ-command-example-prefix: 'sudo -u www-data php occ'
     occ-command-example-prefix-no-sudo: 'php occ'
     php-supported-versions-url: https://secure.php.net/supported-versions.php
-    recommended-php-version: 7.2
+    recommended-php-version: 7.3
     std-port-http: 8080
     std-port-memcache: 11211
     std-port-mysql: 3306
     std-port-redis: 6379
-    supported-php-versions: '7.0, 7.1, 7.2, 7.3'
+    supported-php-versions: '7.2 & 7.3'
```

Comment:
The changes here can be reversed. All of 7.0, 7.1, 7.2, 7.3 are supported (for now, oC  10.3) - even though we might wish to drop support for old PHP versions 7.0 and 7.1, we have not done that just yet.

Reply:
Sorry about that. I misunderstood the discussion earlier about this.

Reply:
The changes to the supported versions has been updated.